### DISCUSSION TITLE
Collecting jmx metrics from zookeeper and kafka

### DISCUSSION THREAD

**Discussion by LORSfrom95** [strimzi/strimzi-kafka-operator#8382] [Q&A]
Hi everyone! I'm trying to configure collecting jmx metrics for my kafka and zookeeper instances (deployed in kubernetes using strimzi). I added "metricsConfig:" and created configmap as in this (https://github.com/strimzi/strimzi-kafka-operator/blob/0.34.0/examples/metrics/kafka-metrics.yaml) example and addet "jmxOptions: {}" to my kafka resource. Is there anything else that must be done to collect jmx metrics? Do I have to manually deploy jmx exporter? Because I don't see them in prometheus (for ex: "zookeeper_quorumsize" query returns empty result)

**Comment by scholzj** [comment]
Sounds like a duplicate to https://cloud-native.slack.com/archives/CMH3Q3SNP/p1681354792965239 I guess.
